manifests/j/JetBrains/Space/2021.1.2.68492/JetBrains.Space.yaml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: JetBrains.Space
+PackageVersion: 2021.1.2.68492
```

Comment:
```suggestion
PackageVersion: 2021.1.2
```

Reply:
That is the actual version name, which is also what is used for the other jetbrains packages

Reply:
The version in the manifest has to match with what's shown in Control Panel.

Reply:
The version in the manifest has to match with what's shown in Control Panel.

Reply:
I see